ci-operator/step-registry/openshift/e2e/test/netobserv-qe/openshift-e2e-test-netobserv-qe-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: openshift-e2e-test-netobserv-qe
+  env:
+  - name: TEST_SCENARIOS
+    default: Network_Observability
+    documentation: |-
+      Select the test case. It's a RE pattern and is passed to the 'grep -E'
```

Comment:
@liangxia - I am not sure if I have got this correct, is this how you'd override?

Reply:
Yes, override the envs in the chain.
We can make it more easier. Currently this PR override the envs in the chain, also copies the test step `openshift-extended-test-web-tests` to `openshift-extended-netobserv-test-web-tests`.  We can reuse the original step ( without copy a new one), if any default env does not fit, we can override in the chain.

Reply:
okay, I re-used the web-tests with additional `elif` condition in [openshift-extended-web-tests-commands.sh](https://github.com/openshift/release/pull/50068/files#diff-0e8a0ad691eb00ba7088443d10dafb3aa558ceb7f73706031b60afa0e9d9940b).